resources/views/people/activities/edit.blade.php

```diff
@@ -32,57 +32,10 @@
-
-              {{-- Summary --}}
-              <div class="form-group">
-                <label for="summary">{{ trans('people.activities_summary') }}</label>
-                <input type="text" class="form-control" name="summary" value="{{ $activity->getSummary() }}"autofocus required maxlength="254">
-              </div>
-
-              {{-- Date --}}
-              <div class="form-group">
-                <label for="specific_date">{{ trans('people.activities_add_date_occured') }}</label>
-                <input type="date" id="specific_date" name="specific_date" class="form-control"
-                    value="{{ $activity->date_it_happened->format('Y-m-d') }}"
-                    min="{{ \Carbon\Carbon::now(Auth::user()->timezone)->subYears(10)->format('Y-m-d') }}"
-                    max="{{ \Carbon\Carbon::now(Auth::user()->timezone)->format('Y-m-d') }}">
-              </div>
-
-              {{-- Build the Activity types dropdown --}}
-              <div class="form-group">
-                <label for="activityType">{{ trans('people.activities_add_pick_activity') }}</label>
-                <select id="activityType" name="activityType" class="form-control" required>
-                  {{-- Blank option --}}
-                  <option value="0" {{ (is_null($activity->activity_type_id)?'selected':'') }}>
-                    -
-                  </option>
-                  @foreach (App\ActivityTypeGroup::all() as $activityTypeGroup)
-                    <optgroup label="{{ trans('people.activity_type_group_'.$activityTypeGroup->key) }}">
-                      @foreach (App\ActivityType::where('activity_type_group_id', $activityTypeGroup->id)->get() as $activityType)
-                      {{ $activity->activity_type_id }}
-                        <option value="{{ $activityType->id }}" {{ ($activity->activity_type_id == $activityType->id)?'selected':'' }}>
-                          {{ trans('people.activity_type_'.$activityType->key) }}
-                        </option>
-                      @endforeach
-                    </optgroup>
-                  @endforeach
-                </select>
-              </div>
-
-              <div class="form-group">
-                <label for="comment">{{ trans('people.activities_add_optional_comment') }}</label>
-                <textarea class="form-control" id="comment" name="comment" rows="3">{{ $activity->getDescription() }}</textarea>
-              </div>
-
-              <div class="form-group actions">
-                <button type="submit" class="btn btn-primary">{{ trans('people.activities_add_cta') }}</button>
-                <a href="/people/{{ $contact->id }}" class="btn btn-secondary">{{ trans('app.cancel') }}</a>
-              </div> <!-- .form-group -->
-            </form>
+            @include('people.activities.form', [
+              'method' => 'PUT',
+              'action' => route('people.activities.update', [$contact, $activity])
```

Comment:
As a brief warning, if #243 is merged before this it will throw errors because the route URLs were not standardised. Some forms posted to `/save` while some posted to `/store`, so the tests I wrote reflected this. Once this is merged I will update the tests to reflect the new URLs.